recipe/ocr_hmm_decoder.patch

```diff
@@ -0,0 +1,11 @@
+--- opencv_contrib/modules/text/src/ocr_hmm_decoder.cpp
++++ opencv_contrib/modules/text/src/ocr_hmm_decoder.cpp
+@@ -982,7 +982,7 @@
+ 
+     nr_feature  = weights.rows;
+     nr_class    = weights.cols;
+-    patch_size  = (int)sqrt(kernels.cols);
++    patch_size  = (int)sqrt((float)kernels.cols);
```

Comment:
[Fixed in latest version](https://github.com/Itseez/opencv_contrib/blob/master/modules/text/src/ocr_hmm_decoder.cpp#L985)